hack/install.sh

```diff
@@ -23,7 +23,15 @@ set -o pipefail
 
 export SCALE_CHAOSDUCK_TO_ZERO=1
 export REPLICAS=1
-export KO_FLAGS=${KO_FLAGS:-"--platform=\"linux/$(uname -m)\""}
+# Map 'uname -m' output to the expected architecture name for 'ko'
+ARCH=$(uname -m)
+case "$ARCH" in
+    x86_64) KO_ARCH="amd64" ;;
+    arm64) KO_ARCH="arm64" ;;
+    *) echo "Unsupported architecture: $ARCH" && exit 1 ;;
+esac
+
+export KO_FLAGS=${KO_FLAGS:-"--platform=linux/$KO_ARCH"}
```

Comment:
Hey @Leo6Leo now that I think about this a bit more, I think maybe a better approach would be to check the architecture from `go env`, as that would match the name of the architecture used by `ko`.

Maybe something like:
```bash
go env -json | jq .GOARCH -r
```

But, this would add a dependency on `jq` :shrug: 

@pierDipi @creydr any ideas/opinions?